vertx-template-engines/vertx-web-templ-rocker/src/main/java/io/vertx/ext/web/templ/rocker/impl/RockerTemplateEngineImpl.java

```diff
@@ -22,29 +22,24 @@
 import io.vertx.core.Future;
 import io.vertx.core.Handler;
 import io.vertx.core.buffer.Buffer;
-import io.vertx.ext.web.common.template.CachingTemplateEngine;
 import io.vertx.ext.web.templ.rocker.RockerTemplateEngine;
 
 import java.util.Map;
 
 /**
  * @author <a href="mailto:xianguang.zhou@outlook.com">Xianguang Zhou</a>
  */
-public class RockerTemplateEngineImpl extends CachingTemplateEngine<Void> implements RockerTemplateEngine {
```

Comment:
Rocker doesn't need caching?

Reply:
No, rocker templates are compiled at `compile` time, caching here is irrelevant as we cache a java object that is always the same reference. This was a "bug" in the previous implementation.